README.md

```diff
@@ -355,16 +355,55 @@ steps:
 [...]
 ```
 
-### `steps[*].retry:` `steps.<key>.retry:`
+### `steps[*].loop:` `steps.<key>.loop:`
```

Comment:
@k2tzumi 
To share the implementation image, I updated the README first.

I would like to hear your opinion.

Reply:
I felt strong feelings 👍 

We believe that even though it will be difficult to integrate with the functionality of the retry: section, we will try to make sure that it is not on the same level of hierarchy as the loop: section.

```suggestion
### `steps[*].[req|greq].retry:` `steps.<key>.[req|greq].retry:`
```

We believe it will be like the above.

Reply:
I use `retry:` for both db runner and exec runner.
For example, it can wait when updating values to the database asynchronously.
It is also possible to use it in the include runner to wait.

Reply:
Sure, there's asynchronous processing.